doc/nrf/ug_contribution.rst

```diff
@@ -0,0 +1,141 @@
+.. _ug_contribution:
+
+Contributing to the |NCS|
+*************************
+
+The |NCS| is maintained by Nordic Semiconductor, but it is open to community contributions.
+
+As a general practice, we encourage to make small, controlled changes.
+This practice simplifies reviewing, merging, and rebasing.
+In addition, it keeps the change history clear and clean.
+
+When contributing to the |NCS|, you should provide as much information as you can about your change.
+Before submitting, you must also update the appropriate documentation and test your changes thoroughly.
+
+The general GitHub workflow that is used by |NCS| developers is based on a combination of Git commands on the command line and browser interaction with GitHub.
+As usual with Git, there are multiple ways of getting a task done.
+The following steps show a typical workflow:
+
+1. :ref:`dm-wf-fork`.
+   The following substeps summarize the most important commands to start working with a fork of the repository:
+
+   a. Fork the sdk-nrf repository to your personal account on GitHub.
+      To do so, click the `fork button on the sdk-nrf repository page`_ (upper right corner).
+
+   #. On your computer, change into the :file:`nrf` folder that was created when you :ref:`obtained the code <cloning_the_repositories_win>`::
+
+         cd ncs/nrf
+
+      Rename the default remote that points to the `upstream nRF Connect SDK repository <sdk-nrf_>`_ from ``origin`` to ``ncs``::
+
+         git remote rename origin ncs
+
+      Let Git know about the fork that you just created and name it ``origin``.
+      Replace *your_github_id* with your GitHub user name.
```

Comment:
let's be consistent, we've used `{username}` in the dev model guide

Reply:
It's wrong in the dev model guide. Using {} or <> to denote variables adds complexity, because the user might not know if these characters should be typed in or not.
Therefore, common doc style is to use italics for variables.